### DISCUSSION TITLE
Who approve changes?

### DISCUSSION THREAD

**Discussion by tijno** [deso-protocol/dips#13] [Q&A]
Who can approve changes ? 

CIP process lists:
- [core devs](https://github.com/orgs/bitclout/people)
- community

Is it worth listing & naming them in CIP so changes to approvers follows CIP process.

**Comment by HPaulson** [comment]
Another note here -- the "Large" section mentions that only two core devs are required to approve a consensus-level change. This seems like it defeats the purpose of CIPs.

Should this not be updated to include community reviews and approvals? I think all PRs across core should absolutely require review and approval from community members -- not just core devs.

**Comment by dgsus** [comment]
while im very happy to see CIP's out, i must say im a bit disappointed on its framework, since there are no votes. 
there is no doing anything without original devs buy in. im hopeful thought that is just a first draft/approach to CIP's. 

a discussion should start on how CIP's should be handled/voted in to make Bitclout a community driven project.

While i do trust the original devs have best intentions in mind, we should not *need* to trust them. CIP's should be a tool for the community to influence the path of the project in trustless way.

i know this is not a simple endeavor... how will people be selected to vote (to avoid bots) , what will be the weight of their vote? hopefully it wont be (only) related to the amount of Clout they are staking.

**Reply by HPaulson** [reply]
Adopting CIPs (and decentralization in general) is a gradient, not a switch. Consensus on these things is very very hard to perfect, especially at an early stage like this one which requires quick-pace by the core team.

Having a simple framework like this one which allows us to have open transparency regarding changes is already a MASSIVE step in the right direction, and we'll improve gradually as time goes on. For now, I think if we work out the minor required approvals, we should be more than fine for the short term.

**Reply by tijno** [reply]
Yep i think what you describe @dgsus is where we want to end up, but as @HPaulson says - its a gradient.

We are moving from a sitation where core team had full control and no community, to in future a CIP model aligned with what you describe.

Step by step. 

Delighted its here so we can get going.

**Comment by omicreativedev** [comment]
I feel as anything involving a fork or major consensus or protocol change should involve the known validated contributing community developers and independent node operators until there is a way to govern through node itself or by stake/whatever (PoS? dPoS? PoA?)
